eng/pipelines/templates/pipeline.yml

```diff
@@ -286,7 +286,7 @@ stages:
       BuildNumber: $[stageDependencies.Initialize.Initialize_Build.outputs['updatebuildnumber.BuildNumber']]
       FullVstsBuildNumber: $[stageDependencies.Initialize.Initialize_Build.outputs['updatebuildnumber.FullVstsBuildNumber']]
     pool:
-      vmImage: windows-latest
+      name: VSEngSS-MicroBuild2022Preview-1ES
```

Comment:
Can you explain this change a bit?  Guessing: we used to care about the latest windows image, but now we care more about whatever is in the MicroBuild Preview pool?